### PR TITLE
Allow to send notifications with persist event

### PR DESCRIPTION
**Pull request by krichprollsch**
in relation with Calendart/CalendArt#8

### CODE REVIEW

File: src/EventApi.php [hunk 1]
```diff
@@ -243,6 +243,17 @@ public function persist(CalendArtAbstractEvent $event)
...
+        // manage the options now
+        // should we send notifications to the attendees ?
+        if (isset($options['sendNotifications'])) {
+            $url = sprintf(
+                '%s?sendNotifications=%s',
+                $url,
+                $options['sendNotifications'] ? 'true' : 'false'
+            );
+
+        }
```
Comment (Taluu) [inline src/EventApi.php:256]: you should add a documentation of some sort in the phpdoc for this option

File: src/EventApi.php [hunk 2]
```diff
@@ -243,6 +243,17 @@ public function persist(CalendArtAbstractEvent $event)
             $method = 'post';
         }
 
+        // manage the options now
+        // should we send notifications to the attendees ?
+        if (isset($options['sendNotifications'])) {
```
Comment (Taluu) [inline src/EventApi.php:254]: you should test if it is true

File: src/EventApi.php [hunk 3]
```diff
@@ -243,6 +243,17 @@ public function persist(CalendArtAbstractEvent $event)
...
+            $url = sprintf(
+                '%s?sendNotifications=%s',
+                $url,
+                $options['sendNotifications'] ? 'true' : 'false'
+            );
```
Comment (Taluu) [inline src/EventApi.php]: not a fan of building the url like this. You should use the guzzle's query object

File: src/EventApi.php (hunk 2 above)
Comment (krichprollsch) [inline src/EventApi.php:254]: I test the value of the option here https://github.com/Calendart/GoogleAdapter/pull/13/files/29f9eb4b0d170924d71304671bd082b877bb332e#diff-1369ed56bd7644f751c9b9838ed842a9R252

Comment (Taluu) [inline src/EventApi.php:254]: yarp so nevermind

**Comment by Taluu** [comment]
:+1:

**Pull request merged**